content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+authors: ["sophia-parafina"]
+tags: [". NET", "C#", "F#", "Visual Basic"]
+meta_image: "microsoft_logo.jpg"
+meta_desc: "Pulumi supports . NET languages for Infastructure as Code"
+date: "2019-11-08"
+---
+
+We are excited to announce the preview of Pulumi . NET support. Pulumi lets you declare infrastructure using general purpose programming languages. Our goal is to automate applications at all layers and help developers and operators work better together by eliminating the silos that separate them.
+
+Millions of . NET developers can now use their favorite languages and open source ecosystems to build modern, cloud native applications. We've added support for C#, F# and Visual Basic. Because the . NET framework is available on Windows, Linux and macOS you have a choice of platforms to use. You can create cloud resources by writing **Microsoft . NET Core** programs to build and deploy AKS Clusters, Functions, Azure App Services, Virtual Machines, Cosmos DBs, KeyVaults, and much, much more.
+
+## What Does Pulumi Enable?
+
+As a . NET developer you can:
+
+* Declare infrastructure in C#, F#, or Visual Basic NET.
+* Code with favorite IDEs, including Visual Studio, Visual Studio Code, and Rider, with auto-completion and refactoring tools.
+* Catch mistakes early on with red markers and build errors.
```

Comment:
Isn't reusing the bits from the .NET blog post a problem? Also, SEO-wise -- might be considered "plagiarism"?

Reply:
rewrote this bit